guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -34,16 +34,16 @@ include::common/modules/proc_configuring-remaining-smart-proxy-servers-with-defa
 
 include::common/modules/con_configuring-smart-proxy-server-with-custom-ssl-certificates-for-load-balancing-without-puppet.adoc[leveloffset=+2]
 
-include::common/modules/proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc[leveloffset=+3]
+:ssl-common-name: {smartproxy-example-com}
+:ProductName: {SmartProxyServer}
+include::common/modules/proc_creating-a-custom-ssl-certificate.adoc[leveloffset=+3]
```

Comment:
```suggestion
include::common/modules/proc_creating-a-custom-ssl-certificate.adoc[leveloffset=+3]
:!ProductName:
```

Unless "ProductName" was set before, then please use the following:

````
:parent-ProductName: {ProductName}
my include ...
:ProductName: {parent-ProductName}
:!parent-ProductName:
````

Reply:
@maximiliankolb if I set these values on line 37, the value it acquires is `Foreman` and not `Smart Proxy`, looks like I have to set it to `{SmartProxyServer}` manually first.

Reply:
No, you should not have to set "SmartProxyServer" anywhere because it's part of guides/common/attributes*.adoc".

Maybe you can do some local testing and see what's the value of "ProductName" is certain points in the docs.

Reply:
So the value is specifically set to `{SmartProxyServer}` in `doc-Installing_Proxy/master.adoc` otherwise it is set to `{ProjectServer}`. The `doc-doc-Configuring_Load_Balancer/master.adoc` file picks the universal value which is `{ProjectServer}`.

I have tweaked the `parent-ProductName` local variable usage and used it a little more flexibly. Does that work?

Reply:
okay looks like it doesn't :(